src/components/card/MyMovie/index.tsx

```diff
@@ -0,0 +1,58 @@
+import { IMyMovieCard } from 'interfaces/card';
+import React from 'react';
+import * as Styles from './styles';
+
+const MyMovie: React.FunctionComponent<IMyMovieCard> = ({
+  id,
+  title,
+  runningtime,
+  time,
+  location,
+  audiences,
+  fare
+}) => {
+  let seatText = ''
```

Comment:
이런건 reduce로 조질수 있는데요?
```suggestion
const seatText = audiences.reduce((acc, cur, index) => {
  const isLast = index === audiences.length - 1;
  return `${acc}${cur}${isLast ? '' : ','}`
}, '');
```

Reply:
reduce가 생소해서 생각도 못했네요~~~~~